File: christmas/december task 2/array tasks other languages/minimum.py
def minimumIn(array, smallest):

    smallest  = array[0]
        
    for count in range (0, len(array)):
        if(array[count] < smallest ):

            smallest = array[count]
            

    return smallest
        
array = [9, 7, 2, 1]

smallest = 0

print(minimumIn(array, smallest))

File: christmas/december task 2/array tasks other languages/Minimum.js
function minimumIn(array, smallest) {

        smallest  = array[0];
        
        

    for(count = 1; count < array.length; count ++) {
        if(array[count] < smallest ) {

            smallest = array[count]; 
            
        }

           
        }

        return smallest;
        
   }

let  array = [9, 7, 2, 1];

smallest = 0;

console.log(minimumIn(array, smallest));
